apps/acrs/behaviours/aggregator-save-update.js

```diff
@@ -1,3 +1,5 @@
+let isUpdate;
+let updatingIndex;
```

Comment:
Can we store these values in session rather than the global js scope?

Reply:
Stored in sessionModel